CodeForPhilly/third-places
# Add initial models

# Overview

Adds the following models in an initial form + generates an initial migration
* Location
* LocationType
* Tag
* TagType
* BusinessHours
* ThirdPlaceUser

Closes ###

#8

# Demo

Docker Containers start up with models and migrations applied.
![2023-05-03_docker_log_intial_models](https://user-images.githubusercontent.com/1811328/235833582-9f75c65f-569c-4e00-a605-093cae88732e.png)

Database Schema after migration:
![Screenshot_20230503_004037](https://user-images.githubusercontent.com/1811328/235834676-22e921d1-2e83-47ed-be08-240e9d8b5989.png)

Location table:
![Screenshot_20230503_004419](https://user-images.githubusercontent.com/1811328/235834724-b32c05a2-ed79-413e-ae6b-a9912b54c8cb.png)

ThirdPlaceUser table:
![Screenshot_20230503_004745](https://user-images.githubusercontent.com/1811328/235834731-3b609ad0-f849-4ea4-a065-695571284c2d.png)


# Testing instructions

- Start up Docker containers, make sure django and postgresql start with no errors.
- Connect to db via
    - docker exec -it <DJANGO_DOCKER_CONTAINER_NAME> bash
    - python3 manage.py dbshell
- Check that corresponding tables to models are in fact created with the appropriate columns.

# Checklist before merge
